ChangeLog.org

```diff
@@ -1,4 +1,16 @@
 * Changelog for Juvix
+** 0.1.0.2
+- _Changes to =Library/Sexp.hs=_
+  + Added documentation to all functions
+  + Changed =Sexp.flatten= to be a full flatten
+  + Fixed a small bug in =Sexp.foldr1= where it behaves poorly with
+    dotted lists
+    * Also created a test to accommodate for this new change
```

Comment:
Words like `also` seem redundant to me and rely on context

Reply:
I'd also avoid using `this` to refer to a previous message

Reply:
How should I mention the bullet this one is the child of?

Reply:
I would avoid nesting. In this case:
```
- Fix: bug in =Sexp.foldr1= on dotted lists
- Test:  =Sexp.foldr1= on dotted lists
- Feat: Add dotted list syntax to the Sexp Show instance
```

Reply:
These are just suggestions. E.g. Not sure if `Test: XXX` is a good idea. Want to know your thoughts as well

Reply:
Semes to me that it just removes any structure that may exist, as now the context between the two is not obvious from a quick look. However if it is nested, then you can assume the nested items refer to the one above it, thus a test on it would intuitively refer to the entry above

Reply:
Could have the nested section just start wtih `Tests` much like what I've just pushed